src/track/heaptrack.sh.cmake

```diff
@@ -94,6 +95,14 @@ while true; do
             usage
             exit 0
             ;;
+        "-o" | "--output-file")
+            output=$2
+            if [ -z "$output" ]; then
+                echo "Missing output argument."
+                exit 1
+            fi
+            shift 2
```

Comment:
We will need to turn the potentially-relative-path into an absolute one, using `readlink -f`. Otherwise, when you inject into an already running application, a relative path will be resolved against the CWD of the application which is highly unexpected to the user. Can you fix that please?